.prettierignore

```diff
@@ -1,10 +1,12 @@
 dist/
 .cache/
 coverage/
+/test*.*
```

Comment:
These are temporary files like `test.js`, `test2.css`, etc. for quick local testing and debugging. Adding them to `.prettierignore` doesn't make sense as it won't be possible to test Prettier on them.

Reply:
🤦